### PR TITLE
Completed the Translation API 

### PR DESCRIPTION
**Pull request by zindzigriffin**
…rMain.

### CODE REVIEW

**Review by calcazaro — APPROVED** [review]

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -7,3 +7,9 @@
 #greeting-container {
   margin-top: 20px;
 }
+#lang {
+        display: block;
```
Comment (zsxking) [inline src/main/webapp/style.css:11]: fix indentation to be consistent with rest of the file

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function requestTranslation() {
+        const phrase = document.getElementById('text').value;
```
Comment (zsxking) [inline src/main/webapp/script.js:30]: fix indentation

**Review by zsxking — APPROVED** [review]
LGTM, only couple styling fix.

**Comment by zsxking** [comment]
fix #3

**Pull request merged**